tests/cases/assetstore_test.py

```diff
@@ -193,12 +188,26 @@ def testFilesystemAssetstoreImport(self):
         file = self.model('file').load(resp.json['_id'], force=True, exc=True)
         self.assertTrue(os.path.isfile(file['path']))
 
-        # Test importing a directory
-        params['importPath'] = os.path.join(
-            ROOT_DIR, 'tests', 'cases', 'py_client')
-        resp = self.request(path, method='POST', params=params, user=self.admin)
+        self.model('file').remove(file)
+
+        # Test importing directory with include & exclude regexes; file should be excluded
+        params['importPath'] = os.path.join(ROOT_DIR, 'tests', 'cases', 'py_client')
+        self.request(
+            path, method='POST', params=dict(
+                params, fileIncludeRegex='world.*', fileExcludeRegex='world.*'))
+        resp = self.request('/resource/lookup', user=self.admin, params={
+            'path': '/user/admin/Public/world.txt/world.txt',
+            'test': True
+        })
+        self.assertStatusOk(resp)
```

Comment:
The coverage report implies that this call isn't doing what we expect (the line where the exclusion filter occurs is reported as uncovered).